chatbot_train/unittest_train.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+import numpy as np
+from train import train_run
+from bert_model import BERT_Arch
+import os
+import sys
+import pandas
+import yaml
+from yaml.loader import SafeLoader
+
+
+class TestTrain(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        cfg_path = os.path.dirname(os.path.abspath(__file__))
+        cfg_file = cfg_path + "/" + "test_config.yaml"
+        self.test_cfg = {}
+        with open(cfg_file) as c_info_file:
+             self.test_cfg = yaml.load(c_info_file, Loader=SafeLoader)
+        self.test_cfg = self.test_cfg["test_arguments"]
+        
+ 
+    def test_default_parms(self):
+            self.assertTrue(train_run())
+            train = train_run()[1]
+            try:
+               self.assertAlmostEqual(train,25,places=1)
+            except AssertionError:
+                pass
+
+    def test_with_parms(self):
+        #setup data
+        data = self.test_cfg["data1"]
+        epochs = self.test_cfg["epochs1"]
+        #test with data and epochs values
+        train = train_run(data,epochs)[1]
+        try:
+           self.assertAlmostEqual(train,25,places=1)
+        except AssertionError:
+            pass
+        self.assertTrue(train_run(data,epochs))
+    
+    def test_negative_epoch(self):
+        #setup data
+        data = self.test_cfg["data2"]
+        epochs = self.test_cfg["epochs2"]
+        self.assertEqual(train_run(data,epochs),False)
```

Comment:
train runs returns tuple ! 
return train_loss, train_accuracy, test_accuracy 

not bool ?